node-stringprep.cc

```diff
@@ -19,13 +19,13 @@ class StringPrep : public ObjectWrap {
 public:
   static void Initialize(Handle<Object> target)
   {
-    NanScope();
-    Local<FunctionTemplate> t = NanNew<FunctionTemplate>(New);
+    Nan::HandleScope scope;
+    Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
     NanAssignPersistent(stringprep_constructor, t);
```

Comment:
`stringprep_constructor` needs to be declared as a `Nan::Persistent` now (instead of just `Persistent`, i.e. `v8::Persistent`) and then you use `stringprep_constructor.Reset(t)` here